keys/keys.go

```diff
@@ -62,7 +59,7 @@ func NodeStatusKey(nodeID int32) proto.Key {
 
 // MakeNameMetadataKey returns the key for the namespace.
 func MakeNameMetadataKey(parentID uint32, name string) proto.Key {
-	k := make([]byte, 0, len(NameMetadataPrefix)+maxVarintSize+len(name))
+	k := make([]byte, 0, len(NameMetadataPrefix)+binary.MaxVarintLen64+len(name))
```

Comment:
I see it being used by Encoding() in 

util/encoding/encoding.go

Should I rename maxVarintSize to MaxEncodeUvarint and export it from util/encoding/encoding.go package then?